C/primitive/elements/jets.c

```diff
@@ -382,6 +421,28 @@ bool output_null_datum(frameItem* dst, frameItem src, const txEnv* env) {
   return true;
 }
 
+/* output_surjection_proof : TWO^32 |- S TWO^256 */
```

Comment:
output_surjection_proof and output_range_proof are wrong and have the wrong types.

The specification requires that they return `S (S TWO^256)`.

So either the specification needs to be changed or the implementation needs to be changed.

Reply:
The specification has been now changed.